cmd/elkey/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Eigenlayer batch key manager"
+	app.Commands = []*cli.Command{
+		commandGenerate,
+	}
+
+	if err := app.Run(os.Args); err != nil {
+		_, err := fmt.Fprintln(os.Stderr, err)
+		if err != nil {
+			return
+		}
+		os.Exit(1)
```

Comment:
Shouldn't this be reversed? You want to `os.Exit(1)` when there's an error and os.Exit(0) (aka return) otherwise

Reply:
this whole thing is in error condition
```
_, err := fmt.Fprintln(os.Stderr, err)
		if err != nil {
			return
		}
		os.Exit(1)
```
but maybe there is a better way to handle.

Reply:
Ah right. Then remove the if err return. You want to always exit with 1 since the app failed.

Reply:
yea removed. and just logged error